Gemfile

```diff
@@ -6,9 +6,7 @@ gem 'jira-ruby', :group => :development
 group :test do
   gem 'rspec'
   gem 'beaker', '~>1.21.0'
-  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '0.12.1'
-  end
+  gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.7")
```

Comment:
Doesn't matter to me either.  @nwolfe - think there's something important in `0.8.0` that we might want / need for Puppet Server eventually anyway?  If not, I'm good sticking with `0.7.~` since we're using it successfully on the other branches already.